graphql_api/types/repository/repository.py

```diff
@@ -582,18 +581,12 @@ def resolve_coverage_analytics(
     )
 
 
-@repository_bindable.field("testResultsAggregates")
-@convert_kwargs_to_snake_case
-async def resolve_test_results_aggregates(
```

Comment:
`testResultsAggregates` and `flakeAggregates` are not yet being used by frontend client so moving them to `testAnalytics` as a breaking change with this PR